### DISCUSSION TITLE
GO organ file

### DISCUSSION THREAD

**Discussion by ChasFW** [GrandOrgue/grandorgue#978] [Help] [answered]
I'm in the process of creating a VPO from our local church organ (1M+P with 10 stops). I have created a nice working odf. I have now started sampling the pipes (4 seconds per sample). I have sampled the 4' Principal into a 32-bit wav file, removed the noise, and extracted each note into a separate wav file. Then I have used wavpack to compress to wv files. All good so far.

When I go to the Virtual Organ interface and play a note, it starts correctly, but goes on playing until the wav file finishes, not when I release the note.

**Clearly I do not understand looping**, and how to make a wav (or wv) file which combines attack and release, and a sound in between when (and only when) the note is held. Please can somebody explain to me how to do this? When I search the internet for looping, it shows how to extend an existing wav file out for a longer period, but this is not what is needed for GO.

Thank you for your help.

**Comment by ChasFW** [comment, marked as answer]
I am able to answer this myself!

Lars Palo's LoopAuditioneer is the answer. Initially I had problems getting this to run due to issues with Norton not liking it, but Norton was appeased and it is now working. It is important to tweak the start/end points in the viewer after the loop has been found. It took me a while to realise that it is essential to crossfade the selected loop, otherwise there will inevitably be clicks when the loop repeats.  I am finding it best to pick about one second or so towards the end of the sample to pick the loop. Then right click the waveform just before the end to activate the release cue marker. At the moment this procedure takes forever. I'm hoping I will get faster as time goes by.

Any tips from seasoned users of this software would be most welcome. I have processed just two pipes!!!

**Reply by sandstone42** [reply]
Loop Auditioneer has a batch process in which it will search for loops of all files in a directory.The cue marker still has to be added manually to each sample. Cross fading in some cases will result in a noticeable fluctuation in volume.

**Reply by ChasFW** [reply]
Thanks.

The organ I'm sampling is not great, and there is much sound fluctuation during the playing of a note. Hence the difficulty of finding good loop points and the need to crossfade. The resulting fluctuation in volume actually well represents the sound the organ makes anyway. The pipes for a given stop themselves are not well matched, and some of the pedals start with a rattle, while others are just wheezy.  That's the charm of the organ! It's unique!

**Reply by paolo2504** [reply]
ChasFW it is interesting what you’re doing. 
I would appreciate a couple of hints:
a. some links to documentation about recording samples for instruments (not necessarily only organs)

b. You wrote you using wavpack. Which your feeling about option -f (fast) vs -h (high) ?
Better quality ? Just Better compression ?..
Regards

**Comment by ChasFW** [comment]
I have a new problem which I don't understand.

I have successfully created three stops for my VPO. I am now getting an issue with the latest stop I have added. The loops are _**usually**_ no longer looping. The only difference is that I have made the loops longer in LoopAuditioneer- e.g. 2 seconds. Could this be the reason? Is there a setting in the ODF to indicate what the maximum loop length can be?

**Reply by ChasFW** [reply]
The problem has "gone away". Not sure what caused it, nor why it went away. I'll blame Windoze.
